### PR TITLE
Feature/#501 banner message for review

### PR DESCRIPTION
**Pull request by tomlovesgithub**

### CODE REVIEW

File: src/views/Exercises/Stages/ShortlistList.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
 </template>
 
 <script>
-import Banner from '../../../components/Page/Banner';
+import Banner from '@/components/Page/Banner';
```
Comment (warrensearle) [inline src/views/Exercises/Stages/ShortlistList.vue:22]: This is redundant in this PR

File: src/components/Page/Banner.vue [hunk 2]
```diff
@@ -25,7 +26,8 @@ export default {
   props: {
     message: {
       type: String,
-      required: true,
+      default: null,
+      // @TODO add validator?
```
Comment (warrensearle) [inline src/components/Page/Banner.vue:30]: Ideally the changes in this file should be in a different PR

Comment (lloback) [reply src/components/Page/Banner.vue]: @tomlovesgithub don't forget that a PR should only contain changes related to the ticket itself. Any refactoriing or other changes should go on a different branch and ticket.
